README.md

```diff
@@ -71,84 +77,78 @@ Now, we use a JSON configuration (Note you should be able to modify to use a dif
 
-Key ```$.config.plugin``` references your Spark application and provides it a name and version as specified in **Step 2**
+Key ```artifact``` in the above JSON specifies the CDAP application template into which your Spark application will be embedded into. 
 
-### Configurations
+### Application Name
 
-Rest of the JSON is specifies the configuration for sending emails in case there is an issue with your Spark Application. You can specify multiple recipents in case there is any issue with your Spark application. 
+Key ```config.plugin``` references your Spark application and provides it a name and version as specified in **Step 2**
 
-> At this point you should be able to go to your CDAP UI and see your Spark application.
+> At this point you should be able to go to your CDAP UI and see the WordCount application.
 
-## Run your Spark application in CDAP
+## Step 4 - Run your Spark application in CDAP
 
-Now, that we have deployed your Spark application, wrapped it with CDAP, it's time to run your CDAP application. This specific example that we have used for illustration expects two arguments as specified [here](https://github.com/caskdata/sample-spark-app/blob/develop/word-count-java/src/main/java/com/example/spark/JavaWordCount.java#L19).
+The Spark application can now be run in CDAP. This specific example that we have used for illustration expects two arguments (input file and output directory) as specified [here](https://github.com/caskdata/sample-spark-app/blob/develop/word-count-java/src/main/java/com/example/spark/JavaWordCount.java#L19).
 
-Following is the REST you can you use to start the application
+The application can be started using the UI or the REST API:
 ```
-curl -w"\n" -X POST "localhost:11015/v3/namespaces/default/apps/WordCountApp/workflows/NotifiableWorkflow/start" \
+curl -w"\n" -X POST "localhost:11015/v3/namespaces/default/apps/WordCount/workflows/NotifiableWorkflow/start" \
      -d '{"program.args": "/data/input.txt /data/output.dir"}'
 ```
 
-Few things to note here
+The status of the application can be monitored through the UI. Once the Spark application finishes running the output will be available in the output directory specified.
 
-* As the CDAP application template is wrapping it in ```NotifiableWorkflow```, you start the workflow. 
-* You pass the runtime argument as POST body.
-* And because it's a workflow it can scheduled to run periodically based on the set schedule. 
+## Advanced Options
 
-> **We are done, it was quick & easy to integrate your existing application into CDAP.**
+Now that the Spark program is running as a CDAP Application, users can leverage the additional capabilities provided by the CDAP platform such as - scheduling, logs, metadata, security and the like. The following section describes scheduling capability. 
 
-## Optional - Working with Schedules - Create, Update & List schedule for your application
+## Working with Schedules - Create, Update & List schedule for your application
```

Comment:
`Working with Schedules - Create, Update, and List schedule for your application`